src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,168 @@
+    </UserControl.Resources>
+
+    <Grid>
+        <StackPanel Margin="4,0,4,4">
+            <TextBlock x:Name="GraphOptionsHeading"
+                       x:Uid="GraphOptionsHeading"
+                       FontSize="22"
+                       AutomationProperties.HeadingLevel="Level1"/>
+            <TextBlock x:Name="GridHeading"
+                       x:Uid="GridHeading"
+                       Margin="0,12,0,6"
+                       Style="{StaticResource SubTitleTextBoxStyle}"
+                       AutomationProperties.HeadingLevel="Level2"/>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="auto"/>
+                    <RowDefinition Height="10"/>
+                    <RowDefinition Height="auto"/>
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="2*"/>
+                    <ColumnDefinition Width="10"/>
+                    <ColumnDefinition Width="2*"/>
+                </Grid.ColumnDefinitions>
+
+                <TextBox x:Name="SettingsXMin"
+                         x:Uid="GraphSettingsXMin"
+                         Style="{x:Bind SelectTextBoxStyle(ViewModel.XError), Mode=OneWay}"
+                         PreviewKeyDown="GridSettingsTextBox_PreviewKeyDown"
+                         Text="{x:Bind ViewModel.XMin, Mode=TwoWay}"/>
+                <TextBox x:Name="SettingsXMax"
+                         x:Uid="GraphSettingsXMax"
+                         Grid.Column="2"
+                         Style="{x:Bind SelectTextBoxStyle(ViewModel.XError), Mode=OneWay}"
+                         PreviewKeyDown="GridSettingsTextBox_PreviewKeyDown"
+                         Text="{x:Bind ViewModel.XMax, Mode=TwoWay}"/>
+
+                <TextBox x:Name="SettingsYMin"
+                         x:Uid="GraphSettingsYMin"
+                         Grid.Row="2"
+                         Style="{x:Bind SelectTextBoxStyle(ViewModel.YError), Mode=OneWay}"
+                         PreviewKeyDown="GridSettingsTextBox_PreviewKeyDown"
+                         Text="{x:Bind ViewModel.YMin, Mode=TwoWay}"/>
+                <TextBox x:Name="SettingsYMax"
+                         x:Uid="GraphSettingsYMax"
+                         Grid.Row="2"
+                         Grid.Column="2"
+                         Style="{x:Bind SelectTextBoxStyle(ViewModel.YError), Mode=OneWay}"
+                         PreviewKeyDown="GridSettingsTextBox_PreviewKeyDown"
+                         Text="{x:Bind ViewModel.YMax, Mode=TwoWay}"/>
+            </Grid>
+
+            <TextBlock x:Name="UnitsHeading"
+                       x:Uid="UnitsHeading"
+                       Margin="0,16,0,6"
+                       Style="{StaticResource SubTitleTextBoxStyle}"
+                       AutomationProperties.HeadingLevel="Level2"/>
+
+            <StackPanel Orientation="Horizontal">
+                <RadioButton x:Name="Radians"
```

Comment:
Changing from Radians/Degrees/Gradians while the graph is empty appears to have no effect and resets the graph to default settings. This causes the values in the y and x min/max to become out of sync with the actual graph values as they also get reset but the textbox has the old values.

Reply:
Workaround added.